RiiConnect24Patcher.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
```

Comment:
this run only on elementary and other distros

Reply:
Hmm... I don't know much about how the first line works, I think it specifies the shell to use, but are you sure this won't break other OS? I guess it'd be fine...